.github/workflows/build-all.yml

```diff
@@ -0,0 +1,67 @@
+###########################################################################
+# GitHub Action Workflow
+# On pull request or changes to scripts/commands or changes to the pipeline
+# code to any branch besides develop, staging and master triggers the full 
+# build and test pipeline.
+# 
+# NOTE: develop, staging(main) and master are skipped on the push because this
+#       would be redundant after running the full set of tests from the PR.
+#       See deploy-develop-on-merge.yml and make_erd for the workflow
+#       pipelines that run on merge to develop, staging, and master branches.
+#
+# Step 0: make PR from your branch into develop, or make changes on your 
+#         branch to non-documentation files in scripts or commands.sh and 
+#         push changes to your remote branch.
+#
+# Step 1: Makes a request to the V2 CircleCI API to initiate the project,
+#         which will filter based upon build_and_test_backend and 
+#         build_and_test_frontend to run the workflow/jobs listed here:
+#           build-and-test:[
+#             test-backend,
+#             test-frontend,
+#             test-e2e
+#           ]
+#
+#         Leverages the open source GitHub Action:
+#           https://github.com/promiseofcake/circleci-trigger-action
+###########################################################################
+name: Build and test All for PRs and when scripts/commands change
+on:
+  push:
+    branches_ignore:
+      - develop
+      - main
+      - master
+    paths:
+      - 'scripts/**'
+      - 'commands.sh'
```

Comment:
I am not sure if we really need to test on changes to commands. We are only using the owasp scans on circleci and we should definitely move these two out of commands.sh since the intention behind commands.sh was only to have scripts for local dev